spec/models/currently_inked_spec.rb

```diff
@@ -73,85 +67,83 @@
   end
 
   describe '#unarchivable?' do
+    let(:pen) { create(:collected_pen, user: user) }
+    let(:twilight) { create(:collected_ink, user: user, ink_name: 'Twilight') }
+    let(:pumpkin) { create(:collected_ink, user: user, ink_name: 'Pumpkin') }
 
-    let(:pen) { collected_pens(:monis_wing_sung) }
-
-    before(:each) do
+    before do
       subject.update!(
         collected_pen: pen,
-        collected_ink: collected_inks(:monis_fire_and_ice),
+        collected_ink: pumpkin,
         archived_on: Date.today
       )
     end
 
     it 'returns false if there is an active currently inked with that pen' do
       user.currently_inkeds.create!(
         collected_pen: pen,
-        collected_ink: collected_inks(:monis_marine)
+        collected_ink: twilight
       )
       expect(subject).to_not be_unarchivable
     end
 
     it 'returns true if there is an archived entry with that pen' do
       user.currently_inkeds.create!(
         collected_pen: pen,
-        collected_ink: collected_inks(:monis_marine),
+        collected_ink: twilight,
         archived_on: Date.today
       )
       expect(subject).to be_unarchivable
     end
   end
 
   describe '#collected_pens_for_active_select' do
+    let(:pen) { create(:collected_pen, user: user) }
+    let(:all_pens) do
+      [
+        pen,
+        create(:collected_pen, user: user, brand: 'Pilot', model: 'Custom 74')
+      ]
+    end
 
-    let(:pen) { collected_pens(:monis_wing_sung) }
+    before { all_pens }
```

Comment:
The idiomatic way to do this is `let!` (same goes for a few other `before` blocks above).

Reply:
Happy to try and remember to do it that way for this project.

I've been used to strictly avoiding `let!` in favor of explicit `before` statements, mainly because I kept flagging rubocop's [LetSetup](https://www.rubydoc.info/gems/rubocop-rspec/1.7.0/RuboCop/Cop/RSpec/LetSetup) rule.  I also just like the idea of all my `let` calls merely being recipes rather than explicit setup, but I know there are tradeoffs (see [discussion about disabling the LetSetup rule by default](https://github.com/rubocop-hq/rubocop-rspec/issues/94), for example).